simple/sample/input/config.json

```diff
@@ -24,6 +24,14 @@
     "wikidataids.csv": {
       "entityType": "Country",
       "provenance": "Provenance1 Name"
+    },
+    "latlng_events.csv": {
+      "importType": "events",
+      "eventType": "CrimeEvent",
+      "entityType": "CensusZipCodeTabulationArea",
```

Comment:
with eventType, the entityType becomes a bit confusing now.. I think this worth some thoughts now.

Reply:
I agree. It's really *location* type. However, we've used `entityType` for observations and will be using it for entities as well so kept the same name for consistency.

I'll try to make sure to clarify this in the config documentation.